src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -59,26 +59,11 @@
       <div class="background-color-light-blue">
         <div class="row">
           <div class="col-12">
-            <div class="row padding-15px text-left">
-              <div class="col-md-2">
-                <div class="question-basic-info">Question</div>
-              </div>
-              <div class="col-md-10">
-                <textarea id="question-brief" class="form-control" [ngModel]="model.questionBrief" (ngModelChange)="triggerModelChange('questionBrief', $event)" [disabled]="!model.isEditable"
-                          placeholder="A concise version of the question e.g. &quot;How well did the team member communicate?&quot;"
-                ></textarea>
-              </div>
-            </div>
-            <div class="row padding-15px text-left">
-              <div class="col-md-2">
-                <div class="question-basic-info">[Optional]<br/>Description</div>
-              </div>
-              <div class="col-md-10">
-                <tm-rich-text-editor [richText]="model.questionDescription" (richTextChange)="triggerModelChange('questionDescription', $event)" [isDisabled]="!model.isEditable" minHeightInPx="100"
-                                     placeholderText="More details about the question e.g. &quot;In answering the question, do consider communications made informally within the team, and formal communications with the instructors and tutors.&quot;"
-                ></tm-rich-text-editor>
-              </div>
-            </div>
+            <tm-question-edit-brief-description-form [brief]="model.questionBrief" (briefChange)="triggerModelChange('questionBrief', $event)" [isBriefDisabled]="!model.isEditable"
+                                                     briefPlaceholder="A concise version of the question e.g. &quot;How well did the team member communicate?&quot;"
+                                                     [descriptionRichText]="model.questionDescription" (descriptionChange)="triggerModelChange('questionDescription', $event)" [isDescriptionDisabled]="!model.isEditable"
+                                                     descriptionPlaceholderText="More details about the question e.g. &quot;In answering the question, do consider communications made informally within the team, and formal communications with the instructors and tutors.&quot;"
```

Comment:
I don't think we need a customizable placeholder text for either brief or description.

Reply:
What if you want customizable placeholders later? My questions is: Why not give different placeholders to differnt types of questions?

Reply:
We do it when we need it. Follow the YAGNI principle.

Reply:
Ok...